assets/js/src/form_editor/components/editor.jsx

```diff
@@ -43,6 +43,10 @@ export default () => {
     (sel) => sel('mailpoet-form-editor').getSidebarOpened(),
     []
   );
+  const isFullscreen = useSelect(
```

Comment:
Now I see that this is used only for changing body class. So I think we could create a component FullscreenMode that would render null but it would contain this selector and effect. We could then just place the component <FullscreenMode /> in editor component.  Editor component is already quite big so this could make it a bit more readable. What do you think?

Reply:
Right, it will be better.